src/acquisition/nidss/taiwan_nidss.py

```diff
@@ -204,11 +203,6 @@ def get_dengue_data(first_week, last_week):
       fields = line.split(',')
       location_b64 = base64.b64encode(fields[3].encode('utf-8'))
       location = NIDSS._TRANSLATED[location_b64]
-      region = NIDSS.LOCATION_TO_REGION[location]
-      imported_b64 = base64.b64encode(fields[6].encode('utf-8'))
-      imported = imported_b64 == b'5piv'
-      sex = fields[5]
-      age = fields[7]
```

Comment:
maybe just comment out the `region`, `sex`, and `age` lines and note that theyre currently unused?  leaving them in could help someone debug later (at least in terms of parsing the input file), and they might even be data points we will add in the future...